statsmodels/stats/contingency_tables.py

```diff
@@ -1055,30 +1126,42 @@ def test_null_odds(self, correction=False):
 
     @cache_readonly
     def oddsratio_pooled(self):
-        # doc for cached attributes in init above
+        """
+        The pooled odds ratio.
 
+        The value is an estimate of a common odds ratio across all of the
+        stratified tables.
+        """
         odds_ratio = np.sum(self._ad / self._n) / np.sum(self._bc / self._n)
         return odds_ratio
 
     @cache_readonly
     def logodds_pooled(self):
```

Comment:
should this also be more explicit and more consistent with log_riskratio_se and `oddsratio_pooled`?
eg. `log_oddsratio_pooled`